ginga/Control.py

```diff
@@ -576,8 +581,8 @@ def  _dl_indicator(count, blksize, totalsize):
             try:
                 dl = catalog.URLServer(self.logger, "downloader", "dl",
                                        info.url, "")
-                filepath = dl.retrieve(info.url, filepath=info.filepath,
-                                       cb_fn=_dl_indicator)
+                dl.retrieve(info.url, filepath=info.filepath,
+                            cb_fn=_dl_indicator)
```

Comment:
I think we need `filepath` below.  So we shouldn't get rid of the assignment here.